assets/style.css

```diff
@@ -88,13 +88,27 @@ h1 {
   border: 1px solid #1a2236;
 }
 
+.Start-collaboration:hover {
+  background-color: #e5e5e5;
+  color: #1a2236;
+}
+
+.Start-collaboration:-moz-suppressed {
+  background-color: #e5e5e5;
+  color: #1a2236;
+}
```

Comment:
- [ ] Great job trying to implement the hover and pressed styles for your buttons. However, I noticed that the background color of the buttons when on hover and pressed is different from what is in the original design :+1: 
- In your design, you have this color
`background-color: #e5e5e5;`
instead of
`background-color: #fff;`

| Your Design on Hover | The Figma Design On Hover |
|--------|--------|
| ![image](https://github.com/axcell-0/Thierrys-portfolio/assets/102699958/409482ed-4122-46ef-bbd4-33c0f1296820) | ![image](https://github.com/axcell-0/Thierrys-portfolio/assets/102699958/6b892aca-0174-450a-982f-88924ebc4ab4) | 

- If you refer to the example the previous reviewer gave you in this [comment](https://github.com/axcell-0/Thierrys-portfolio/pull/3#discussion_r1219775352) You can see that he put the right colors in the example for you. So kindly refer to that :wink: 
- [ ] I also noticed that the `pressed state` doesn't have the border width increasing as seen in the Figma design and in the example the previous reviewer gave you. So kindly fix that as well :wink:  
- Please fix the background color for the pressed state as well :wink: 

| Your Design when Pressed | The Figma Design when Pressed |
|--------|--------|
| ![image](https://github.com/axcell-0/Thierrys-portfolio/assets/102699958/ba584e08-315c-4704-94ed-c4310b4d19e6)| ![image](https://github.com/axcell-0/Thierrys-portfolio/assets/102699958/32403620-9b9e-4c3d-98f6-ad7c34b61a1b) |

Reply:
thanks but for the pressed if I can't afford to put it can the next reviewer help me in that cause I tried but it was not functioning thanks in advance

Reply:
@axcell-0, I have provided a guide below. Kindly read through and implement. If you are still stuck then you can contact me on zoom to discus your approach.